source/sections/_lose_the_spreadsheet.html.erb

```diff
@@ -21,6 +21,10 @@
             <%= image_tag "icons/#{col.icon}.svg", class: "icon" %>
             <h3 class="heading"><%= col.title %></h3>
             <p><%= col.description %></p>
+            <br />
+            <a href="<%= localize_path(col.link, locale_id: locale_obj[:id]) %>" title="Learn More" class="featured_link__learn-more btn--solid btn--secondary">
```

Comment:
I may have given you a bad suggestion here—I believe we use `localize_url` for *everything*—looking at `view-source:https://staging-www.sharesight.com/ca/` I only see `href="https://staging-www.sharesight.com/…`..

```suggestion
            <a href="<%= localize_url(col.link, locale_id: locale_obj[:id]) %>" title="Learn More" class="featured_link__learn-more btn--solid btn--secondary">
```